### PR TITLE
Issue #5791: wrap lines at .github and some files root of repo

### PR DESCRIPTION
**Pull request by romani**
Issue #5791

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -16,7 +16,8 @@
 [[PLACE YOUR OUTPUT HERE]]
 ```
 
-`/var/tmp $ java -Duser.language=en -Duser.country=US -jar checkstyle-X.XX-all.jar -c config.xml YOUR_FILE.java`:
+`/var/tmp $ java -Duser.language=en -Duser.country=US
```
Comment (rnveach) [inline .github/ISSUE_TEMPLATE.md]: doesn't this need something like `\` to say it is a line continuation?

Comment (romani) [reply .github/ISSUE_TEMPLATE.md]: hmm, what about such template ?:
```
/var/tmp $ javac YOUR_FILE.java
#[[PLACE YOUR OUTPUT HERE]]

/var/tmp $ cat YOUR_FILE.java
#[[PLACE YOU OUTPUT HERE]]

/var/tmp $ cat config.xml
#[[PLACE YOUR OUTPUT HERE]]

/var/tmp $ RUN_LOCALE="-Duser.language=en -Duser.country=US"
/var/tmp $ java $RUN_LOCALE -jar checkstyle-X.XX-all.jar -c config.xml YOUR_FILE.java
#[[PLACE YOUR OUTPUT HERE]]
```

image

Comment (rnveach) [reply .github/ISSUE_TEMPLATE.md]: I am fine with either way you want to go.

Comment (romani) [reply .github/ISSUE_TEMPLATE.md]: updated

**Pull request closed without merging**